src/main/groovy/com/google/protobuf/gradle/ProtobufPlugin.groovy

```diff
@@ -53,42 +62,65 @@ class ProtobufPlugin implements Plugin<Project> {
     }
 
     void apply(final Project project) {
-        this.project = project
-        def gv = project.gradle.gradleVersion =~ "(\\d*)\\.(\\d*).*"
-        if (!gv || !gv.matches() || gv.group(1).toInteger() < 2 || gv.group(2).toInteger() < 2) {
-            println("You are using Gradle ${project.gradle.gradleVersion}: This version of the protobuf plugin requires minimum Gradle version 2.2")
+      this.project = project
+      def gv = project.gradle.gradleVersion =~ "(\\d*)\\.(\\d*).*"
+      if (!gv || !gv.matches() || gv.group(1).toInteger() < 2 || gv.group(2).toInteger() < 2) {
+          println("You are using Gradle ${project.gradle.gradleVersion}: This version of the protobuf plugin requires minimum Gradle version 2.2")
+      }
```

Comment:
I know the indentation is inconsistent everywhere, but please keep the format as is and let the commit contain only the meaty content. Formatting should be addressed as a separate commit.